### DISCUSSION TITLE
Suggestion for test schedulers

### DISCUSSION THREAD

**Discussion by JeffVM-Solutions** [seleniumbase/SeleniumBase#925] [Ideas]
I would like to start by saying this is an awesome framework! 

I am using the framework to create UI tests as a monitoring tool for web apps (Synthetic transaction monitoring). I was wondering if you have a suggestion on a python/docker friendly scheduler where I can run the tests on a consistent basis? And, Cron Jobs or Windows Task Manager are not good options.

**Comment by mdmintz** [comment]
@DevOps-Martins , Hello, and thank you! I run most of my tests (in the corporate world) in Jenkins. It's easy to create a new Jenkins job that runs on a schedule. Jenkins is open source, and you can run that for free if you already have infrastructure to house it. If you want cloud hosting, I found a way to do it as cheaply as $13.61/month for closed source projects (https://github.com/seleniumbase/SeleniumBase/blob/master/integrations/google_cloud/ReadMe.md). If the project is open source, you can probably run it for free on a schedule with GitHub Actions (https://github.com/seleniumbase/SeleniumBase/actions), which is what I use for verifying SeleniumBase builds for releases. There is a Dockerfile if you need that (https://github.com/seleniumbase/SeleniumBase/blob/master/Dockerfile) but it's mainly used for setting up a container for SeleniumBase, and you'd still need to add a tool in there for scheduling runs.

**Reply by JeffVM-Solutions** [reply]
Awesome! Thank you so much! I plan on creating this app all in a containerized environment. The scripts will be be running on a 5 minutes interval, which should give me a decent MTTR if something does go down. I'm also going to use Gitlab for CI/CD of the app itself. This should be a very interesting project. Thank you again for your assistance.
